spec/features/pom_caseload_feature_spec.rb

```diff
@@ -129,11 +145,14 @@
       end
     end
 
-    it 'can be sorted by location' do
+    it 'can be sorted by cell location' do
+      # Rotl's offenders are grouped when sorted by cell location. If sorted in ascending order they appear
+      # at the top otherwise they are grouped at the bottom
```

Comment:
In fact you could probably even use this comment ^^
It'd just be nice to have it explained in the implementation, too.